filament/backend/src/opengl/PlatformCocoaGL.mm

```diff
@@ -76,8 +76,10 @@
         [NSNotificationCenter defaultCenter] addObserverForName: NSViewGlobalFrameDidChangeNotification
         object: nil
         queue: nil
-        usingBlock: ^ (NSNotification * note) {
-            [nsOpenGLContext update];
+        usingBlock: ^ (NSNotification* note) {
+            if (note.object == pImpl->mCurrentView) {
```

Comment:
Dot instead of arrow because Objective C.  :)

Reply:
Wait this is potentially the wrong mCurrentView, let me tweak this by adding a copy to the closure...

Reply:
Actually that's somewhat non-trivial, I'll need to move the observer creation into makeCurrent 😭

Reply:
Fixed this up.  Should be ready for review now.